[2024-01-12T02:53:55.404+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: rds_test_3.send_slack_postAndReview manual__2024-01-12T02:53:36.592129+00:00 [queued]>
[2024-01-12T02:53:55.470+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: rds_test_3.send_slack_postAndReview manual__2024-01-12T02:53:36.592129+00:00 [queued]>
[2024-01-12T02:53:55.471+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-01-12T02:53:55.473+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-01-12T02:53:55.474+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-01-12T02:53:55.563+0000] {taskinstance.py:1300} INFO - Executing <Task(SlackWebhookOperator): send_slack_postAndReview> on 2024-01-12 02:53:36.592129+00:00
[2024-01-12T02:53:55.580+0000] {standard_task_runner.py:55} INFO - Started process 136 to run task
[2024-01-12T02:53:55.594+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'rds_test_3', 'send_slack_postAndReview', 'manual__2024-01-12T02:53:36.592129+00:00', '--job-id', '172', '--raw', '--subdir', 'DAGS_FOLDER/rds_test_3.py', '--cfg-path', '/tmp/tmp4546u9pc']
[2024-01-12T02:53:55.596+0000] {standard_task_runner.py:83} INFO - Job 172: Subtask send_slack_postAndReview
[2024-01-12T02:53:55.964+0000] {task_command.py:388} INFO - Running <TaskInstance: rds_test_3.send_slack_postAndReview manual__2024-01-12T02:53:36.592129+00:00 [running]> on host 3d6f932ce29c
[2024-01-12T02:53:56.392+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=rds_test_3
AIRFLOW_CTX_TASK_ID=send_slack_postAndReview
AIRFLOW_CTX_EXECUTION_DATE=2024-01-12T02:53:36.592129+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-12T02:53:36.592129+00:00
[2024-01-12T02:53:56.397+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/providers/slack/operators/slack_webhook.py:173: UserWarning: Found unexpected keyword-argument(s) 'link_names' in `send` method. This argument(s) have no effect.
  link_names=self.link_names,

[2024-01-12T02:53:56.432+0000] {base.py:73} INFO - Using connection ID 'slack_conn' for task execution.
[2024-01-12T02:53:56.442+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/providers/slack/hooks/slack_webhook.py:226: DeprecationWarning: Found Slack Webhook Token URL in Connection 'slack_conn' `host` and `password` field is empty. This behaviour deprecated and could expose you token in the UI and will be removed in a future releases.
  return WebhookClient(**self._get_conn_params())

[2024-01-12T02:53:56.962+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=rds_test_3, task_id=send_slack_postAndReview, execution_date=20240112T025336, start_date=20240112T025355, end_date=20240112T025356
[2024-01-12T02:53:57.079+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-01-12T02:53:57.317+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
